client/cli/lib/build.ts

```diff
@@ -16,44 +16,51 @@ export const enum Mode {
 
 export interface Options {
   mode?: Mode;
-  file?: string;
+  dir?: string;
   modifyRollupConfig?: ExtendRollupConfig;
 }
 
-export default async ({ mode = Mode.development, file = getMainFile(), modifyRollupConfig }: Options = {}): Promise<
-  any
-> => {
+export default async ({ mode = Mode.development, dir, modifyRollupConfig }: Options = {}): Promise<any> => {
```

Comment:
how about adding a default value for dir? `dir = '.'` like you got in mode? wdyt?

Reply:
JS is weird, if you pass undefined it won't use the default value, it will pass default value only if you didn't set a value at all